openwisp_firmware_upgrader/migrations/0003_default_permissions.py

```diff
@@ -0,0 +1,52 @@
+from django.db import migrations
+from django.contrib.auth.models import Permission
+
+from . import create_default_permissions
+
+
+
+def assign_permissions_to_groups(apps, schema_editor):
```

Comment:
a similar migration should be created for the sample app as well or we'll have two different test environments.

Since this operation will have to be done not only by the sample_app but potentially by any app that will extend this module, it's good practice to make this function reusable by passing `app_label` as a parameter.

The actual function passed to `RunPython` will call the reusable function.